Service/RequestService.php

```diff
@@ -374,7 +374,8 @@ public function requestHandler(array $data, array $configuration): Response
         // todo: controlleren of de gebruiker ingelogd is
 
         // Make a list of schema's that are allowed for this endpoint
-        $allowedSchemas = [];
+        $allowedSchemas['id'] = [];
+        $allowedSchemas['name'] = [];
```

Comment:
This change is already present on CoreBundle version-kiss. i am not sure why it shows here as if it isn't

Reply:
Same for all other changes in this file